src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -1,17 +1,46 @@
 package juja.sqlcmd;
 
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
 public class DatabaseManagerTest {
+    private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
+    private static final String DB_NAME = "sqlcmd";
+    private static final String DB_USER = "sqlcmd";
+    private static final String DB_USER_PASSWORD = "sqlcmd";
+
+    private static Connection connection;
+
     private DatabaseManager databaseManager;
 
+    @BeforeClass
+    public static void setConnection() throws SQLException {
+        connection = DriverManager.getConnection(DB_CONNECTION_URL + DB_NAME, DB_USER, DB_USER_PASSWORD);
+        dropAllTables();
+    }
+
+    @AfterClass
+    public static void closeConnection() throws SQLException {
+        connection.close();
```

Comment:
may be add check for connection == null?

Reply:
There's no need in this check, because if connection will be null than it's unexpected behavior and it's ok application to crash. In this case all test wouldn't pass.

Reply:
It's tests. It's OK